task-2/src/components/Navigation/index.js

```diff
@@ -11,7 +12,9 @@ export const Navigation = ({ setOpenModal }) => {
 		<div className='navigationContainer'>
 			<p style={{ color: '#F65261' }}>netflixroulette</p>
 			<div className='btnContainer'>
-				<SearchButton />
+				<Link to='/'>
+					<SearchButton />
```

Comment:
Couple of points:
- take a look inside SearchButton - you already have a "<Link>" there, wrapping SearchButton in Link again is redundant an prone to errors
- refactor you button component completely - make it just 1 component that accepts "text" as a prop and an optional "url" as prop (if "url" is truthy, wrap button contents in a Link, if falsy - render just button contents)
- go through the up and reuse your Button component where possible

Reply:
resolved.